frontend/packages/console-shared/src/components/quick-search/QuickSearchList.tsx

```diff
@@ -114,7 +114,9 @@ const QuickSearchList: React.FC<QuickSearchListProps> = ({
                   className="ocs-quick-search-list__item-content"
                   dataListCells={[
                     <DataListCell isIcon key={`${item.uid}-icon`}>
-                      {item.icon?.node ?? getIcon(item)}
+                      <div className="ocs-quick-search-list__item-icon">
```

Comment:
if item.icon? is not present we render empty div. Is this ok? or should we check condition and render the div??

Reply:
In that case, we render the icon by `getIcon(item)}`